ansible/README.md

```diff
@@ -0,0 +1,84 @@
+Deploying Openwhisk using Ansible
+=========
+
+
+### Getting started
+
+If you want to deploy Openwhisk locally using Ansible, you first need to install Ansible on your development environment:
+
+#### Ubuntu users
+```
+sudo apt-get install python-pip
+sudo pip install ansible==2.0.2.0
+```
+
+#### Vagrant users
+Nothing to be done, Ansible is already installed during vagrant provisioning.
+
+#### Mac users
+It is assumed that a VM has been provisioned using **docker-machine**.
+
+```
+brew install python
+pip install ansible==2.0.2.0
+
+cd ansible/environments/mac
+./genHosts.sh <name of docker-machine>
+```
+After this there should be a `hosts` file in the `ansible/environments/mac` dir.
+
+To verify the hosts file you can do a quick ping to the docker machine:
+```
+ansible all -i ansible/enviroments/mac -m ping
+```
+Should result in something like:
+```
+ansible | SUCCESS => {
+    "changed": false,
+    "ping": "pong"
+}
+192.168.99.100 | SUCCESS => {
+    "changed": false,
+    "ping": "pong"
+}
+```
+
+### Install Prerequisites
+This needs to be done only once per environment.
+```
```

Comment:
yes, for now, we support only 2 environments: `local` (vagrant box, execute ansible inside the box) and `mac` (virtualbox, execute ansible in mac). But you're rite, we should describe it more explicitly.